### ISSUE TITLE
Scheduler reset

### ISSUE THREAD

**Issue by HamidrezaKmK** [vahidzee/lightning-utils#1]
When you interrupt the .fit, the learning rate will reset again. 
You would expect it to store information related to learning_rate in the checkpoints.
